docs/source/contents/upgrading/index.md

```diff
@@ -1,5 +1,9 @@
 # Upgrading
 
+## Upgrading from 2.6 - 2.7
+
+If users are using [Prometheus with podmonitors](../../../../prometheus/monitors), they should be aware that all pods provisioned through the `SeldonRuntime` operator will now have the label `app.kubernetes.io/name` for identifying the pods. Previously, the app label has been inconsistent across different versions of Seldon Core v2 (e.g. previously, `app` or a mixture of `app` and `app.kubernetes.io/name` labels was used), so users should modify/configure their existing podmonitors accordingly. 
```

Comment:
:question: 

> they should be aware that all pods provisioned through the `SeldonRuntime` operator will now have the label ... 

Can pods be provisioned not through SeldonRuntime operator? Is that the case when running in non-cluster wide mode? Would they then have the same labels or different ones?